index.html

```diff
@@ -11,97 +12,103 @@
-      (using CHAPI's <code>get()</code> under the hood).
-    </p>
-
-    <p>When you click the Request button, a Verifiable Credential will
-      be requested from your digital wallet.
-    </p>
-
-    <a class="waves-effect waves-light btn" id="requestButton">Present a Credential</a>
-  </div>
+  <div class="container">
+    <div class="card-panel">
+      <h5>Credential Handler API (CHAPI) Demo Verifier</h5>
+      <p><a href="https://github.com/credential-handler/credential-handler-polyfill">Read more about CHAPI</a></p>
+      <p>
+        If you have not yet picked a wallet and registered it with your browser,
+        try out the
+        <a href="https://wallet.example.chapi.io/">Demo Wallet</a>.
+      </p>
+
+      <p>This is a minimal credential Verifier application that demonstrates
+        how an app would ask the user for a credential
+        (using CHAPI's <code>get()</code> under the hood).
+      </p>
 
-  <div class="card-panel hide" id="resultsPanel">
-    <h6>Result of get() operation:</h6>
+      <p>When you click the Request button, a Verifiable Credential will
+        be requested from your digital wallet.
+      </p>
 
-    <pre><code id="getResults"></code></pre>
+      <a class="waves-effect waves-light btn" id="requestButton">Present a Credential</a>
+    </div>
+
+    <div class="card-panel hide" id="resultsPanel">
+      <h6>Result of get() operation:</h6>
+
+      <pre><code id="getResults"></code></pre>
+    </div>
   </div>
-</div>
-<script>
-  async function onClickRequest() {
-    //   document.getElementById('getResults').innerHTML = ''; // clear results
-      const credentialQuery =   {
-        "web": {
-          "VerifiablePresentation": {
-            "query": [{
-              "type": "QueryByExample",
-              "credentialQuery": {
-                "reason": "Please present a UniversityDegreeCredential for JaneDoe.",
-                "example": {
-                  "@context": [
-                    "https://w3id.org/credentials/v1",
-                    "https://www.w3.org/2018/credentials/examples/v1"
-                  ],
-                  "type": ["UniversityDegreeCredential"],
-                  "credentialSubject": {
-                    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
-                  }
-                }
+  <script>
+    async function onClickRequest() {
+      //Create Verifiable Presentation Request
+      const testVpr = {
+        "query": [{
+          "type": "QueryByExample",
+          "credentialQuery": {
+            "reason": "Please present your University Degree to continue the teacher application process.",
+            "example": {
+              "@context": [
+                "https://w3id.org/credentials/v1",
+                "https://www.w3.org/2018/credentials/examples/v1"
+              ],
+              "type": ["UniversityDegreeCredential"],
+              "credentialSubject": {
+                "id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
```

Comment:
Should be able to remove a lot of the quotes here. See the PR in the chapi.io repo for more details

Reply:
See 309367330662c6057534b0436ae67f68a24894a0